qa/L0_unit_tests/test_config_generator.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Also, I think it would be better to keep the `count_tests.py` file because this file doesn't seem to "generate" a unit test. It is only checking the results.

Reply:
Agree